events/room-booking.md

```diff
@@ -17,13 +17,20 @@ Steps for room and AV (Audio Visual) cell booking for conducting workshops and o
 - In order to conduct a workshop during this period, explicit written permission is required from CDC to use a particular room.
 - This permission can only be obtained with the approval of a professor.
 - If a professor agrees to approve the booking, they will have to send an email to CDC (`tnp-off@hijli.iitkgp.ac.in`) with us (`admin@kossiitkgp.org`) and `tpn-placecom@hijli.iitkgp.ac.in` in cc, asking for permission to conduct the event on our behalf.
-- Further steps to be documented.
+- The permission will be emailed back and should be kept show to the guard on the day of the event.
+- The [AV Cell booking](#steps-for-booking-av-audio-visual-cell) booking should be done separately throught the normal procedure. The permission email might be asked for during the AV cell booking process.
```

Comment:
This is a general point, for all venue booking process. Don't mention it just under the CDC times section.

Reply:
> This is a general point, for all venue booking process. Don't mention it just under the CDC times section.

Better to mention it explicitly here since the room booking process is different during CDC. I'll mention the same point in the non-CDC section as well.

Reply:
yes works..